packages/app/src/cli/services/dev.ts

```diff
@@ -167,10 +166,8 @@ async function dev(options: DevOptions) {
     hostname: exposedUrl,
   }
 
-  const [previewableExtensions, nonPreviewableExtensions] = partition(
-    localApp.extensions.ui,
-    (ext) => ext.isPreviewable,
-  )
+  const previewableExtensions = localApp.allExtensions.filter((ext) => ext.isPreviewable)
+  const nonPreviewableExtensions = localApp.allExtensions.filter((ext) => ext.isDraftable)
```

Comment:
Should we change the name of this constant and the functions below form `nonPreviewable` to `draftable`?

Reply:
I renamed every reference of `nonPreviewable` to `draftable`